noir/Earthfile

```diff
@@ -26,6 +25,9 @@ nargo:
 
 packages:
     FROM node:20
+
+    COPY ../barretenberg/ts/+build/build ../barretenberg/ts
```

Comment:
@ludamad Not sure if this is considered hacky.. as `barretenberg/ts` isn't under the same `build` folder. I was having trouble getting resolution to find `barretenberg/ts` through a symbolic link so I just copied one directory back (as `noir-repo` is tested once inside of `noir`) and got the earthly to work.

Reply:
Just mitigated this by specifying with `/build..` and an extra comment